_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
I see your point there. I don't want to sound too technical here while describing a _feature_ but it's possible that _fresh_ is not a real feature of PWA. Maybe what is a feature is the control over the life cycle. With this in mind, lets try again. What about this?

> In the traditional web model, your users are never prompted about installing a new version: as soon as you update the servers, your users get the update. With Progressive Web Applications you are in full control of the web app life-cycle, you decide if you want a silent background update or if you prefer to let the user choose by their own.

Actually, this can be enriched by another user story:

> As a user, I prefer to keep the actual version and not to update immediately.

Perhaps it is against other use cases but it is definitively a valid use case and very related to life-cycle customization.

What do you think @mykmelez ?